### PR TITLE
fix application properties for embedded csp

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
This PR provides a way to define two different headers enforce and report for content security policy delivered through embedded tomcat.

#### Related Pull Requests
* https://github.com/LabKey/server/pull/626

#### Changes
* new application.properties - csp.enforce and csp.report

### CODE REVIEW

**Review by labkey-matthewb — APPROVED** [review]

**Review by labkey-jeckels — APPROVED** [review]

File: server/configs/application.properties [hunk 1]
```diff
@@ -78,10 +78,8 @@ spring.main.banner-mode=off
 # example usage 1 - very strict, disallows 'external' websites, disallows unsafe-inline, but only reports violations (does not enforce)
 # good for test automation!
 
-#csp.disposition=report
-#csp.policy="default-src 'self' https: ;\nconnect-src 'self' https: ;\nobject-src 'none' ;\nstyle-src 'self' https: 'unsafe-inline' ;\nimg-src 'self' data: ;\nscript-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}';\nbase-uri 'self' ;\nupgrade-insecure-requests ;\nframe-ancestors 'self' ;\nreport-to /labkey/admin-contentsecuritypolicyreport.api ;\nreport-uri /labkey/admin-contentsecuritypolicyreport.api ;"
+csp.report="default-src 'self' https: ;\nconnect-src 'self' https: ;\nobject-src 'none' ;\nstyle-src 'self' https: 'unsafe-inline' ;\nimg-src 'self' data: ;\nscript-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}';\nbase-uri 'self' ;\nupgrade-insecure-requests ;\nframe-ancestors 'self' ;\nreport-to /labkey/admin-contentsecuritypolicyreport.api ;\nreport-uri /labkey/admin-contentsecuritypolicyreport.api ;"
```
Comment (labkey-tchad) [inline server/configs/application.properties:81]: There souldn't be a `/labkey` context path for embedded tomcat.
```suggestion
csp.report="default-src 'self' https: ;\nconnect-src 'self' https: ;\nobject-src 'none' ;\nstyle-src 'self' https: 'unsafe-inline' ;\nimg-src 'self' data: ;\nscript-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}';\nbase-uri 'self' ;\nupgrade-insecure-requests ;\nframe-ancestors 'self' ;\nreport-to /admin-contentsecuritypolicyreport.api ;\nreport-uri /admin-contentsecuritypolicyreport.api ;"
```

Comment (labkey-matthewb) [reply server/configs/application.properties:81]: I misread this, I thought this was still commented out.  This should actually point to https://www.labkey.org/admin-contentsecuritypolicyreport.api

Comment (ankurjuneja) [reply server/configs/application.properties:81]: @labkey-matthewb created the following PR https://github.com/LabKey/server/pull/638

**Comment by labkey-tchad** [comment]
@ankurjuneja @labkey-matthewb It looks like this change broke the existing CSP we're using with standalone Tomcat on TeamCity.

**Comment by labkey-tchad** [comment]
> @ankurjuneja @labkey-matthewb It looks like this change broke the existing CSP we're using with standalone Tomcat on TeamCity.

Nevermind. It didn't.

**Pull request merged**